packages/esm-patient-labs-app/src/test-results/panel-view/panel.component.tsx

```diff
@@ -90,15 +90,12 @@ const LabSetPanel: React.FC<LabSetPanelProps> = ({ panel, observations, activePa
   );
 
   return (
-    <Layer>
-      <div
-        className={classNames(styles.labSetPanel, {
-          [styles.activePanel]: activePanel?.conceptUuid === panel.conceptUuid,
-        })}
-        onClick={() => setActivePanel(panel)}
-        role="button"
-        tabIndex={0}
-      >
+    <Layer
+      className={classNames(styles.labSetPanel, {
+        [styles.activePanel]: activePanel?.conceptUuid === panel.conceptUuid,
+      })}
```

Comment:
Moving this styling to the parent container fixes another visual artifact:

![CleanShot 2024-07-31 at 12  59 28@2x](https://github.com/user-attachments/assets/71430208-d88c-425b-b834-33839355c593)